frontend/vre/record/record.list.view.js

```diff
@@ -1,61 +1,68 @@
-import _ from 'lodash';
-import { CollectionView } from 'backbone-fractal';
-import { RecordListItemView } from './record.list.item.view';
-import recordListTemplate from './record.list.view.mustache';
+import Backbone from "backbone";
+import {TabulatorFull as Tabulator} from "tabulator-tables";
+import {properties} from "../utils/record-ontology";
+import {getStringLiteral} from "../utils/jsonld.model";
+import {vreChannel} from "../radio";
 
-export var RecordListView = CollectionView.extend({
-    tagName: 'table',
-    className: 'table table-striped',
-    template: recordListTemplate,
-    container: 'tbody',
+export var RecordListView = Backbone.View.extend({
+    tagName: "div",
```

Comment:
For your information: `'div'` is the default, so you don't strictly need to set the `tagName` when you choose this value. It does not hurt to set it, though.